### PR TITLE
Add mpv.net version 4.1

### LINKED ISSUE #2187: [Request] mpv.net
libmpv based media player. looks like mpv but it has context menu and settings page.

[https://github.com/stax76/mpv.net](https://github.com/stax76/mpv.net)

with release 3.4 it has portable version.

### PR DESCRIPTION
**Pull request by hoilc**
fix #2187

### CODE REVIEW

File: bucket/mpv.net.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            "mpvnet.exe",
+            "mpv.net"
+        ]
+    ],
+    "persist": "portable_config",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/stax76/mpv.net/releases/download/$version/mpv.net-portable-x64-$version.0.0.7z"
```
Comment (Ash258) [inline bucket/mpv.net.json]: Autoupdate will not work for some releases:

A

There is need to go with something like this:

```json
"checkver": {
  "github": "repositoryURL.com",
  "regex": "download/(?<main>\\d.]+)/mpv.net-portable-x64-(?<full>[\\d.]+)\\.7z",
  "replace": "${main}" // or $main, not sure
}, autoupdate with $matchMain and $matchFull
```

Comment (hoilc) [reply bucket/mpv.net.json]: Thank you for suggestion. Full version number is used instead.

File: bucket/mpv.net.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://github.com/stax76/mpv.net/",
+    "description": "A modern media player for Windows that works just like mpv",
+    "license": "MIT",
+    "version": "4.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/stax76/mpv.net/releases/download/4.0/mpv.net-portable-x64-4.0.0.0.7z",
```
Comment (rasa) [inline bucket/mpv.net.json]: Shouldn’t the version be 4.0.0.0, not just 4.0?

Comment (hoilc) [reply bucket/mpv.net.json]: As suggested by Ash258, the main version number is enough.😄️

**Pull request merged**